### ISSUE TITLE
As an author, I want to delete my own posts.

### ISSUE THREAD

**Issue by varsis** [grepme/cmput410-project#8]

**Comment by morganpatz** [comment]
So I feel like I'm so close to getting this to work, but I don't think the database is updating when I delete a post because I'm getting the error "no such table: posts_post_tags" from the line 62 in /posts/views.py

If anyone knows how to fix this, I'm working on branch "deleletePost (I realize my horrible spelling)". If not, I'll look at it tomorrow morning. I think I've been staring at it for too long. 

Note for myself:
Need to add check user functionality to make sure that only the user can delete their posts

**Comment by grepme** [comment]
If there is ever a table missing, Django never synced anything right. Multiple migrations can stack on each other. In this case, it was adding the tags relationship to the table never propagated. 

I don't know why you were replacing hyphens with underscores in the Post model to begin with, was it because of the matching? You had the wrong regex for the url match. You were only matching a to z and not including the hyphen, I updated the syntax and removed the replace string. Also, I synced the database so you don't have to perform the migrations

If you run into similar issues, remove ALL migration folders under each app then run makemigrations and syncdb. 

See commit 0b8e227 for more details
